index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
 
 console.log('Welcome to the calculator!');
 
-console.log('Please enter some input:');
-const response = readline.prompt();
-console.log('You entered: ' + response);
+console.log('Enter the operator:');
+const operator = readline.prompt();
+
+console.log(`How many numbers do you want to ${operator} ?`);
+const c = parseInt(readline.prompt());
```

Comment:
While it doesn't matter in such a short file as this, it's good to get into the habit of giving variables descriptive names. That way when you come back to some code after a break it'll be easier to understand what's going on, especially if it's a more complex program with many, many variables. Instead of `c` you could call this variable something like `numberOfArguments`, it's up to you

Reply:
What if 'parseInt' throws an error?